radio/src/targets/common/arm/stm32/bootloader/boot.cpp

```diff
@@ -509,16 +511,27 @@ int main()
     }
 
     if (state == ST_REBOOT) {
-      lcdClear();
-      lcdRefresh();
-      lcdRefreshWait();
 
 #if defined(RTC_BACKUP_RAM)
       rtcInit();
       RTC->BKP0R = SOFTRESET_REQUEST;
 #endif
 
+      #if defined(RADIO_Commando8)
+      lcdClear();
+      lcdDrawText(2, 22,"Press the power button.");
+      lcdDrawText(2, 33,"Exit the flashing mode.");
```

Comment:
Why should this be specific to this radio? Either it is some new feature / improvement, which should be justified separately and applied to all similar radios, or it shall not be here at all. In general, UI code should not be included directly in the generic bootloader code, as it is common to all radios, with completely different screen sizes, orientation, etc.

Reply:
Because our power switch is controlled by a STM32F031. After entering the BootLoader and exiting opentx through the UI interface, the STMF031 still thinks that opentx is in BootLoader mode. At this time, pressing the power button will cause the STM32F031 to shut down, so you cannot exit through the exit option of the BootLoader UI interface. We'd need to guide the user to press the power button to exit the BootLoader mode. But in order to not affect the general code, we added the COMMANDO8 definition to the BootLoader. Please advice, thanks!